<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookDetailDialog</class>
 <widget class="QWidget" name="BookDetailDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Book Details</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex"><number>0</number></property>
     <widget class="QWidget" name="generalTab">
      <attribute name="title"><string>General</string></attribute>
      <layout class="QVBoxLayout" name="generalLayout">
       <item>
        <widget class="QGroupBox" name="basicInfoGroup">
         <property name="title">
          <string>Basic Information</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_nameEdit">
            <property name="text">
             <string>Name:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="nameEdit">
            <property name="placeholderText">
             <string>Enter book name</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_descriptionEdit">
            <property name="text">
             <string>Description:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="descriptionEdit">
            <property name="placeholderText">
             <string>Enter description (optional)</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_ledgerCcyCombo">
            <property name="text">
             <string>Currency:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="ledgerCcyCombo">
            <property name="editable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_glAccountRefEdit">
            <property name="text">
             <string>GL Account Ref:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="glAccountRefEdit">
            <property name="placeholderText">
             <string>Enter GL account reference</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_costCentreEdit">
            <property name="text">
             <string>Cost Centre:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="costCenterEdit">
            <property name="placeholderText">
             <string>Enter cost centre</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_bookStatusCombo">
            <property name="text">
             <string>Status:</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QComboBox" name="bookStatusCombo"/>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_bookTypeCombo">
            <property name="text">
             <string>Book Type:</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QComboBox" name="bookTypeCombo">
            <item>
             <property name="text">
              <string>Banking</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Trading</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="provenanceTab">
      <attribute name="title"><string>Provenance</string></attribute>
      <layout class="QVBoxLayout" name="provenanceTabLayout">
       <item>
        <widget class="ores::qt::ProvenanceWidget" name="provenanceWidget" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="deleteButton">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>Save</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ores::qt::ProvenanceWidget</class>
   <extends>QWidget</extends>
   <header>ores.qt/ProvenanceWidget.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
